hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -372,6 +374,11 @@ public boolean hasTargetArtifactAssigned(final long targetId, final String sha1H
         return actionRepository.getById(actionId);
     }
 
+    @Override
+    public List<Action> getActiveActionsByExternalRef(@NotNull final List<String> externalRefs) {
+        return actionRepository.findByExternalRefInAndActive(externalRefs, true);
```

Comment:
additional parameter boolean can be substituted with `findByExternalRefInAndActiveIsTrue`

Reply:
Since it is a repository call, it would be flexible option to keep findByExternalRefInAndActive so that we can query for both active and inactive actions without having to write 2 methods.

Reply:
After discussing over a call, we decided to keep it as it is.